agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -618,3 +618,45 @@ func TestNetworkModeBridge(t *testing.T) {
 	err := networkModeTest(t, agent, "bridge")
 	require.NoError(t, err, "Networking mode bridge testing failed")
 }
+
+// TestCustomAttributesWithMaxOptions tests the ECS_INSTANCE_ATTRIBUTES
+// upon agent registration with maximum number of supported key, value pairs
+func TestCustomAttributesWithMaxOptions(t *testing.T) {
```

Comment:
yeah, there will only be a small change to make it work on windows, can you do that in the same PR?